doc/source/whatsnew/v2.2.0.rst

```diff
@@ -432,6 +432,7 @@ I/O
 Period
 ^^^^^^
 - Bug in :class:`Period` addition silently wrapping around instead of raising ``OverflowError`` (:issue:`55503`)
+- Bug in incorrectly allowing construction of :class:`Period` with certain offsets which don't have an attribute _period_dtype_code (:issue:`55785`)
```

Comment:
I think we're giving a better error message here, rather than fixing a bug - could we rephrase to reflect that, and put this in "other enhancements"?

something like
```
- Improved error message when constructing :class:`Period` with invalid offsets such as "QS" (:issue:`55785`)
```

Reply:
agree, I did as you suggested.